classes/person.rb

```diff
@@ -1,17 +1,18 @@
 require_relative 'nameable'
 
 class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] As required for the project, please make sure that `Getters` and `Setters` are present for `@book` and `@person` instance variables. 
Using `getter` and `setter` methods to access and modify the value of a variable provides a controlled and predictable interface for other objects to interact with the variable, while still maintaining the principles of encapsulation and data hiding. i.e.
```ruby
    ...
    attr_accessor :name, :age, :rentals
    attr_reader :id
    ...
```

Reply:
Updated 🆗